### PR TITLE
D1 billing observability

### PR DESCRIPTION
**Pull request by ShaheenFattoe8**
Added a Billing Observability section to the D1 documentation and updated the pricing page with links to address questions regarding billing alerts and usage billable analytics.

### CODE REVIEW

File: content/d1/platform/pricing.md [hunk 1]
```diff
@@ -72,3 +72,19 @@ Yes, you can use an index to reduce the number of rows read by a query. [Creatin
 - Does a freshly created database, and/or an empty table with no rows, contribute to my storage?
 
 Yes, although minimal. An empty table consumes at least a few kilobytes, based on the number of columns (table width) in the table. An empty database consumes approximately 100 KB of storage.
+
+- How can I set up billing notifications for my Cloudflare D1 usage?
```
Comment (joshthoward) [inline content/d1/platform/pricing.md]: I am not a fan of documentation which describes how to walk through a web UI. If it's needed the UI is poorly designed (and I don't think it is). It is also burdensome to maintain. I would delete this.

File: content/d1/observability/d1-billing.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+D1 offers billing metrics regarding rows read, rows written, and total storage usage across all databases in your account.
+
+The metrics displayed in the [Cloudflare dashboard](https://dash.cloudflare.com/) are sourced from Cloudflare’s [GraphQL Analytics API](/analytics/graphql-api/). You can access the metrics [programmatically](#query-via-the-graphql-api) via GraphQL or HTTP client.
+
+## View metrics in the dashboard
+
+Total account billable usage analytics for D1 are available in the Cloudflare dashboard. To view current and past metrics for an account:
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
```
Comment (joshthoward) [inline content/d1/observability/d1-billing.md]: Same comment as previous.

File: content/d1/observability/d1-billing.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+Note that billable usage history is stored for a maximum of 30 days.
+
+## Billing Notifications
+
+Billing notifications for D1 are available for users looking to monitor their total account usage. The following notifications are available:
+- Rows Read
+- Rows Written
+- Total Storage
+
+To set up these notifications:
```
Comment (joshthoward) [inline content/d1/observability/d1-billing.md]: Same comment as previous.

**Review by joshthoward — CHANGES_REQUESTED** [review]
In general I would leave this as highlighting the features within the dashboard and linking customers to relevant launch points. Most of the edits to wording I could take or leave.

File: content/d1/platform/pricing.md (hunk 1 above)
Comment (ShaheenFattoe8) [reply content/d1/platform/pricing.md]: Makes sense, my reasoning was to maintain consistency with how other docs pages have done it - Load balancing docs do it similarly [here](https://developers.cloudflare.com/learning-paths/load-balancing/setup/next-steps/#usage-based-notifications)

File: src/content/docs/d1/observability/billing.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+pcx_content_type: concept
+title: Billing
+weight: 10
```
Comment (vy-ton) [inline src/content/docs/d1/observability/billing.mdx]: Page should be after `Observability > Metrics and analytics`

**Review by vy-ton — APPROVED** [review]

File: content/d1/platform/pricing.md (hunk 1 above)
Comment (kodster28) [reply content/d1/platform/pricing.md]: > I am not a fan of documentation which describes how to walk through a web UI. If it's needed the UI is poorly designed (and I don't think it is). It is also burdensome to maintain. I would delete this.

Howdy @joshthoward, I actually used to be in that boat myself... but you'd be surprised at how often we get requests / feedback around this, especially for newer users and double especially when it touches areas of the UI you aren't in all the time.

The other bit I'd mention is that the maintenance cost here should be less as @Oxyjun has started as a technical writer in this space. He'll be able to help both with these updates + more broadly thinking about how we make stuff like this more maintainable. 

Happy to chat internally if you'd like more context.

Comment (vy-ton) [reply content/d1/platform/pricing.md]: @Oxyjun Can you take over this PR and help get it across the line?

File: src/content/docs/d1/observability/billing.mdx (hunk 4 above)
Comment (Oxyjun) [inline src/content/docs/d1/observability/billing.mdx]:
```suggestion
sidebar:
  order: 11
```

File: src/content/docs/d1/observability/billing.mdx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+# Billing
```
Comment (Oxyjun) [inline src/content/docs/d1/observability/billing.mdx]:
```suggestion

```

**Review by Oxyjun — APPROVED** [review]

**Review by Oxyjun — APPROVED** [review]

**Review by Oxyjun — APPROVED** [review]

**Review by joshthoward — APPROVED** [review]

**Pull request merged**